docs/testplan/snappi-tests/capacity_test.md

```diff
@@ -0,0 +1,91 @@
+
+![overview](./capacity_test_diagram.png)
+
+## Label Structure
+
+The collected metrics are structured in the database using two sets of labels
+
+### Common labels for all metrics
+
+These labels are shared across all metrics within one test job and must be included with every metric.
+
+     | Label                     | Example Value  |
+     | ------------------------- | -------------- |
+     | `METRIC_LABEL_TESTBED`    | TB-XYZ         |
+     | `METRIC_LABEL_TEST_BUILD` | 2024.1103      |
+     | `METRIC_LABEL_TEST_CASE`  | mock-case      |
+     | `METRIC_LABEL_TEST_FILE`  | mock-test.py   |
+     | `METRIC_LABEL_TEST_JOBID` | 2024_1225_0621 |
+
+### Metric labels
+
+These labels identify the specific device and component from which a metric is collected.
+
+#### PSU Metrics
+
+     The `show platform psu` command is used on the switch to retrieve PSU metrics. The following labels are expected to be provided:
+
+     | Label                            | Example Value   |
+     | -------------------------------- | --------------- |
+     | `METRIC_LABEL_DEVICE_ID`         | switch-A        |
+     | `METRIC_LABEL_DEVICE_PSU_ID`     | PSU 1           |
+     | `METRIC_LABEL_DEVICE_PSU_MODEL`  | PWR-ABCD        |
+     | `METRIC_LABEL_DEVICE_PSU_SERIAL` | 1Z011010112349Q |
+     | `METRIC_LABEL_DEVICE_PSU_HW_REV` | 02.00           |
+
+#### Sensor Temperature Metrics
+
+     The `show platform temperature` command is used on the switch to retrieve sensor temperatuer metrics. Among the outputs, the "CPU temp sensor" and "Switch Card temp sensor" are of particular interest. The following labels are expected to be provided:
+
+     | Label                           | Example Value   |
+     | ------------------------------- | --------------- |
+     | `METRIC_LABEL_DEVICE_ID`        | switch-A        |
+     | `METRIC_LABEL_DEVICE_SENSOR_ID` | Cpu temp sensor |
+
+#### Queue Metrics
+
+     The `show queue watermark` is used on the switch to retrieve queue metrics. The following labels are expected to be provided:
+
+     | Label                            | Example Value |
+     | -------------------------------- | ------------- |
+     | `METRIC_LABEL_DEVICE_ID`         | switch-A      |
+     | `METRIC_LABEL_DEVICE_PORT_ID`    | Ethernet8     |
+     | `METRIC_LABEL_DEVICE_QUEUE_ID`   | 1             |
+     | `METRIC_LABEL_DEVICE_QUEUE_CAST` | multicast     |
+
+#### Interface Metrics
+
+     The `show interface counters` is used on the switch to retrieve interface metrics. The outputs include drop counters. The following labels are expected to be provided:
+
+     | Label                         | Example Value |
+     | ----------------------------- | ------------- |
+     | `METRIC_LABEL_DEVICE_ID`      | switch-A      |
+     | `METRIC_LABEL_DEVICE_PORT_ID` | Ethernet8     |
```

Comment:
Hi Mei, do you mind to also define the metric value name here and in the telemetry interface?